src/FSharp.Control.TaskSeq.Test/TaskSeq.TakeWhile.Tests.fs

```diff
@@ -206,3 +226,33 @@ module SideEffects =
 
         repeat |> should not' (equal expected)
     }
+
+module Other =
+    [<Theory>]
+    [<InlineData(false, false)>]
```

Comment:
that's now begging to be replaced with a `MemberData(nameof variants)`
```
let variants : obj[] =
    [| false; false
       .... |]       
```
And at that point I know you'll do a struct typesafe enum for that!

Reply:
Yeah, I considered it. I already have a cleanup branch, will take that there, so as not to block it more. Afaik, an `obj[]` is not valid metadata in attributes, but I can turn it into `ClassData`. Anyway, I agree, some more clarity can be useful, but let's think that through for all the other tests as well (where we mainly just use copies instead of variants).